Docs/source/How_To/coding-style.rst

```diff
@@ -110,7 +157,18 @@ it should be prefixed with the ``Task_`` prefix. the task's name should be in Ca
 
     EEPROM.c //interface with EEPROM
     Temperature.c
-    Task_TaskName.c
+
+Core
```

Comment:
Why not just call this the task layer instead of creating a new name?

Reply:
It is similar to the FreeRTOS/STM cube code generator so if we want to port it over it wouldn't be too much of a hassle.

Reply:
I think "core" is broader and less descriptive than "task" or "tasks" though, since I think we are just going to put tasks in this layer